### DISCUSSION TITLE
Model type not match

### DISCUSSION THREAD

**Discussion by kaiwenxiao** [prisma/prisma#24236] [Q&A] [answered]
### Question

I have a model called artical, which type is generated by prisma client,
using it likes this:

```js
// child component A
import { Artical } from "@prisma/client";
interface ArticalProps {
  data: Artical | null
}

// parent component
const artical = await prisma.artical.findUnique({
    where: {
      id: params.id,
    },
  });
return <A data={artical}/>
```

but I get the type error likes:
```
Prisma__ArticalClient<$Result.GetResult<Prisma.$CompanionPayload<$Extensions.DefaultArgs>, {   where: {     id: string   } }, "findUnique"> | null, null, $Extensions.DefaultArgs>
```
is not assignable to type Artical

```
 Prisma__ArticalClient<GetFindResult<Prisma.$CompanionPayload<$Extensions.DefaultArgs>, {   where: {     id: string   } }> | null, null, $Extensions.DefaultArgs>
```

How can I fix this issue without add as any,
Thanks!

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @kaiwenxiao 👋 

Thank you for raising this question.

Can you please provide a reproducible repository so that we can take a look at this issue?

**Reply by kaiwenxiao** [reply, marked as answer]
Hi @ludralph 
Thanks for your passion, I figure out it myself, just turn on the TS Language Services in my editor WebStorm, and the issue was fixed.

**Reply by edulecca** [reply]
> Hi @ludralph Thanks for your passion, I figure out it myself, just turn on the TS Language Services in my editor WebStorm, and the issue was fixed.

> urn on the TS Language Services

ty works for me!, almost a day trying to figure it out 🤦‍♂️
